com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/SetBreakpointsRequestHandler.java

```diff
@@ -86,7 +86,15 @@ public void initialize(IDebugAdapterContext context) {
         if (context.getDebugSession() == null) {
             return AdapterUtils.createAsyncErrorResponse(response, ErrorCode.EMPTY_DEBUG_SESSION, "Empty debug session.");
         }
-
```

Comment:
In nodebug mode, VS Code Debug Adapter still send setBreakpoints request to the debugger, looks like a bug, can you confirm with their team?